### PR TITLE
Updated command-line usage and systemd integration

### PR DESCRIPTION
**Pull request by maxolasersquad**
## Subject

Some of the English in the cli documentation reads a little awkward. The documentation about integrating with supervisord is incomplete and requires prior knowledge on how to create systemd services.

I am targeting this branch, because it is the latest and relevant to these changes.

### CODE REVIEW

File: docs/reference/command_line.rst [hunk 1]
```diff
@@ -32,6 +33,20 @@ Supervisor is a client/server system that allows its users to monitor and contro
     stdout_logfile=/home/org.sonata-project.demo/logs/sonata_notification.log
     stdout_logfile_maxbytes=10MB
 
+Enable the service to start automatically on boot with the following command as root.
+
+.. code-block:: sh
+
+    systemctl entable sonata_notification
```
Comment (greg0ire) [inline docs/reference/command_line.rst]:
```suggestion
    systemctl enable sonata_notification
```

**Comment by maxolasersquad** [comment]
I wasn't in the right head-space confusing systemd and supervisord. :confused: 

I updated the documentation as appropriate.

File: docs/reference/command_line.rst [hunk 2]
```diff
@@ -11,27 +11,36 @@ This command must be started in production environment to limit memory usage pro
...
-    [program:sonata_production_sonata_notification]
+    [program:yourapp_sonata_notification]
     command=/home/org.sonata-project.demo/current/bin/console sonata:notification:start --env=notification --iteration=250
     autorestart=true
     user=www-data
     redirect_stderr=false
     stdout_logfile=/home/org.sonata-project.demo/logs/sonata_notification.log
     stdout_logfile_maxbytes=10MB
 
+You can check on the status , stop, and start the service with the following respective commands. Read the `supervisor documentation <http://supervisord.org/running.html>`_ for more details.
```
Comment (OskarStark) [inline docs/reference/command_line.rst]: Please remove the whitespace after status

Comment (maxolasersquad) [reply docs/reference/command_line.rst]: Good catch. It has been removed.

**Comment by maxolasersquad** [comment]
Any chance someone could give me feedback or approve this PR?

**Comment by jordisala1991** [comment]
Sorry for the delay, looks ok to me. Lets have another check and we merge it.

**Comment by jordisala1991** [comment]
Thank you @maxolasersquad

**Pull request merged**